File: test/fixtures/java-unirest/get_with_env_var.java
import java.lang.System;
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) {
            HttpResponse<String> httpResponse
               = Unirest.get("http://localhost:28139/v2/images?type=distribution")
            .header("Content-Type", "application/json")
            .header("Authorization", "Bearer " + System.getenv("DO_API_TOKEN"))
            .asString();
    }
}

File: test/fixtures/kotlin-ktor/get_with_env_var.kt
import io.ktor.client.HttpClient
import io.ktor.client.request.get
import io.ktor.client.request.header
import io.ktor.client.statement.bodyAsText
import io.ktor.http.isSuccess
import java.lang.System
import kotlinx.coroutines.runBlocking

val client = HttpClient()

runBlocking {
    val response = client.get("http://localhost:28139/v2/images?type=distribution") {
        header("Content-Type", "application/json")
        header("Authorization", "Bearer " + System.getenv("DO_API_TOKEN") ?: "")
    }
    if (!response.status.isSuccess()) throw Exception("Request failed with status code ${response.status}")

    println(response.bodyAsText())
}
